application/src/main/java/org/togetherjava/tjbot/config/StarboardConfig.java

```diff
@@ -12,15 +12,25 @@ public final class StarboardConfig {
     private final String starboardChannelName;
 
     @JsonCreator(mode = JsonCreator.Mode.PROPERTIES)
-    public StarboardConfig(@JsonProperty(value = "starboardEmojiNames", required = true) List<String> emojiNames,
-                           @JsonProperty(value = "starboardChannelName", required = true) String starboardChannelName) {
+    public StarboardConfig(@JsonProperty(value = "emojiNames", required = true) List<String> emojiNames,
+                           @JsonProperty(value = "channelName", required = true) String starboardChannelName) {
         this.emojiNames = emojiNames;
         this.starboardChannelName = starboardChannelName;
     }
 
+    /**
+     * Gets the names of the emojis whose users react with for it to be put on the starboard
+     *
+     * @return The names of the emojis whose users react with for it to be put on the starboard
+     * */
     public List<String> getEmojiNames() {
         return emojiNames;
     }
+    /**
+     * Gets the name of the channel with the starboard
```

Comment:
does it need to be exact or just a pattern? what is a starboard? what if i leave this empty or put bullshit in it? will it crash or deactivate the feature?...

Reply:
Currently we have it as name but we could do a pattern also.
I am thinking maybe we can do a pattern and that way there can be a starboard in multiple channels?
Not sure of the use case though.

Reply:
i dont mind. my point was rather that the javadoc should answer those questions.